### ISSUE TITLE
Upgrade Ktlint default version

### ISSUE THREAD

**Issue by juherr** [autostyle/autostyle#27]
[The Ktlint default version is currently `0.36.0`](https://github.com/autostyle/autostyle/blob/master/lib/src/main/java/com/github/autostyle/kotlin/KtLintStep.java#L42) when the latest version is `0.41.0`.

**Comment by vlsi** [comment]
Just in case, the version should be overridable from the configuration like `ktlint("0.41.0")`

**Comment by juherr** [comment]
Yes, it is what I did but the style issue wasn't obvious at the first reading.
And the root cause was just the ktlint version which doesn't support the new kotlin versions (and trailing commas).
